annotators/emotional_bot_response/server.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import os
+import sentry_sdk
+import time
+import json
+from os import getenv
+
+from flask import Flask, request, jsonify
+from healthcheck import HealthCheck
+from sentry_sdk.integrations.flask import FlaskIntegration
+
+from common.prompts import (
+    send_request_to_prompted_generative_service,
+    compose_sending_variables,
+)
+
+
+sentry_sdk.init(dsn=os.getenv("SENTRY_DSN"), integrations=[FlaskIntegration()])
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+health = HealthCheck(app, "/healthcheck")
+logging.getLogger("werkzeug").setLevel("WARNING")
+
+GENERATIVE_SERVICE_URL = getenv("GENERATIVE_SERVICE_URL", "http://openai-api-chatgpt:8145/respond")
+GENERATIVE_SERVICE_CONFIG = getenv("GENERATIVE_SERVICE_CONFIG", "openai-chatgpt.json")
+if GENERATIVE_SERVICE_CONFIG:
+    with open(f"common/generative_configs/{GENERATIVE_SERVICE_CONFIG}", "r") as f:
+        GENERATIVE_SERVICE_CONFIG = json.load(f)
+GENERATIVE_TIMEOUT = int(getenv("GENERATIVE_TIMEOUT", 30))
+N_UTTERANCES_CONTEXT = int(getenv("N_UTTERANCES_CONTEXT", 1))
+
+ENVVARS_TO_SEND = getenv("ENVVARS_TO_SEND", None)
+ENVVARS_TO_SEND = [] if ENVVARS_TO_SEND is None else ENVVARS_TO_SEND.split(",")
+
+available_variables = {f"{var}": getenv(var, None) for var in ENVVARS_TO_SEND}
+
+API_CONFIGS = getenv("API_CONFIGS", None)
+API_CONFIGS = [] if API_CONFIGS is None else API_CONFIGS.split(",")
+api_conf = {}
+for config in API_CONFIGS:
+    with open(f"api_configs/{config}", "r") as f:
+        conf = json.load(f)
+        api_conf.update(conf)
+
+for key, value in api_conf.copy().items():
+    for api_key in value["keys"]:
+        if not available_variables[api_key]:
+            del api_conf[key]
+            break
+
+logger.info(f"Available APIs: {', '.join([api['display_name'] for api in api_conf.values()])}")
```

Comment:
you do not use API_CONFIGS and api_conf, so I suggest to remove them (and from yml files too)

Reply:
removed